lib/matplotlib/backends/backend_ps.py

```diff
@@ -802,7 +802,7 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
                 else:
                     kern = 0
                 lastgind = gind
-                thisx += kern/64.0
+                thisx -= kern/64.0
```

Comment:
I'm looking into this further...  I may have been mistaken about this change...  (But in any event the sign does match because the analogous change was made in backend_pdf.py)